eng/scripts/docs/rex-tool-tests.json

```diff
@@ -0,0 +1,30 @@
\src\\docs-ref-autogen\\azure-cosmos-spark_3-1_2-12\\dependency\\stax2-api-3.1.4.jar;D:\\a\\1\\s\\target_repo\\src\\docs-ref-autogen\\azure-cosmos-spark_3-1_2-12\\dependency\\stream-2.9.6.jar;D:\\a\\1\\s\\target_repo\\src\\docs-ref-autogen\\azure-cosmos-spark_3-1_2-12\\dependency\\threeten-extra-1.5.0.jar;D:\\a\\1\\s\\target_repo\\src\\docs-ref-autogen\\azure-cosmos-spark_3-1_2-12\\dependency\\token-provider-1.0.1.jar;D:\\a\\1\\s\\target_repo\\src\\docs-ref-autogen\\azure-cosmos-spark_3-1_2-12\\dependency\\univocity-parsers-2.9.1.jar;D:\\a\\1\\s\\target_repo\\src\\docs-ref-autogen\\azure-cosmos-spark_3-1_2-12\\dependency\\unused-1.0.0.jar;D:\\a\\1\\s\\target_repo\\src\\docs-ref-autogen\\azure-cosmos-spark_3-1_2-12\\dependency\\woodstox-core-5.0.3.jar;D:\\a\\1\\s\\target_repo\\src\\docs-ref-autogen\\azure-cosmos-spark_3-1_2-12\\dependency\\xbean-asm7-shaded-4.15.jar;D:\\a\\1\\s\\target_repo\\src\\docs-ref-autogen\\azure-cosmos-spark_3-1_2-12\\dependency\\xz-1.5.jar;D:\\a\\1\\s\\target_repo\\src\\docs-ref-autogen\\azure-cosmos-spark_3-1_2-12\\dependency\\zookeeper-3.4.14.jar;D:\\a\\1\\s\\target_repo\\src\\docs-ref-autogen\\azure-cosmos-spark_3-1_2-12\\dependency\\zstd-jni-1.4.8-1.jar; -encoding UTF-8 -docletpath D:\\a\\1\\s\\_all\\docfx-doclet.jar -doclet com.microsoft.doclet.DocFxDoclet -sourcepath D:\\a\\1\\s\\target_repo\\src\\docs-ref-autogen\\azure-cosmos-spark_3-1_2-12 -outputpath D:\\a\\1\\s\\target_repo\\src\\docs-ref-autogen\\azure-cosmos-spark_3-1_2-12\\_javadocs -excludePackages :.*implementation.* @files' failed in directory 'D:\\a\\1\\s\\target_repo\\src\\docs-ref-autogen\\azure-cosmos-spark_3-1_2-12' with exit code 2: \nSTDOUT:'Doclet using locale: en_US\n1 error\n\n'\nSTDERR:\n'javadoc: error - No modules, packages or classes specified.\n\n'\n   at Microsoft.Content.Build.Java2Yaml.ProcessUtility.Execute(String fileName, String commandLineArgs, String cwd, Boolean redirectOutput) in C:\\__w\\1\\s\\src\\java2yaml\\Common\\Utility\\ProcessUtility.cs:line 53\n   at System.Threading.Tasks.Task.Execute()\n--- End of stack trace from previous location where exception was thrown ---\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\n   at Microsoft.Content.Build.Java2Yaml.StepCollection.<RunAsync>d__4.MoveNext() in C:\\__w\\1\\s\\src\\java2yaml\\Steps\\StepCollection.cs:line 41\n--- End of stack trace from previous location where exception was thrown ---\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\n   at Microsoft.Content.Build.Java2Yaml.GenerateDocument.<>c__DisplayClass2_0.<<RunAsync>b__0>d.MoveNext() in C:\\__w\\1\\s\\src\\java2yaml\\Steps\\GenerateDocument.cs:line 13\n--- End of stack trace from previous location where exception was thrown ---\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\n   at Microsoft.Content.Build.Java2Yaml.StepCollection.<RunAsync>d__4.MoveNext() in C:\\__w\\1\\s\\src\\java2yaml\\Steps\\StepCollection.cs:line 28\nFailed in 79482 milliseconds.\n"
```

Comment:
Do we really want or need this large comment?

Reply:
We can remove this test case. Those packages don't publish docs anyway.